### PR TITLE
Set params type according to API

### PR DESCRIPTION
**Pull request by rnetser**

### CODE REVIEW

File: ocm_python_wrapper/cluster.py [hunk 1]
```diff
@@ -471,6 +471,39 @@ def __init__(self, client, cluster_name, addon_name):
     def addon_info(self):
         return self.client.api_clusters_mgmt_v1_addons_addon_id_get(self.addon_name).to_dict()
 
+    def get_addon_parameters_dict(self, _addon_parameters):
```
Comment (myakove) [inline ocm_python_wrapper/cluster.py:474]: any reason that `_addon_parameters` is private?

File: ocm_python_wrapper/cluster.py [hunk 2]
```diff
@@ -492,72 +525,61 @@ def validate_and_update_addon_parameters(self, user_parameters=None, use_api_def
...
-        for param, param_dict in required_parameters.items():
-            if param not in user_addon_parameters:
-                if use_api_defaults and param_dict["default_value"]:
+    @staticmethod
+    def update_param_value_type(_user_parameters, addon_parameters_dict):
+        for param in _user_parameters:
+            param_type = addon_parameters_dict[param["id"]]["value_type"]
+            param_value = param["value"]
+            if not isinstance(param_value, param_type):
+                param["value"] = param_type(param_value)  # noqa: FCN001
```
Comment (myakove) [inline ocm_python_wrapper/cluster.py:556]: please remove `noqa: FCN001` and add to exclude list

**Review by myakove — CHANGES_REQUESTED** [review]

File: ocm_python_wrapper/cluster.py (hunk 1 above)
Comment (rnetser) [reply ocm_python_wrapper/cluster.py:474]: no, changed

File: ocm_python_wrapper/cluster.py (hunk 2 above)
Comment (rnetser) [reply ocm_python_wrapper/cluster.py:556]: i do not think it will be clear there

**Review by myakove — APPROVED** [review]

**Pull request merged**